plugins/Terminal/js/components/commandhistorylist.js

```diff
@@ -0,0 +1,41 @@
+import React, { PropTypes } from 'react'
+import { List } from 'immutable'
+
+const CommandHistoryList = ({commandHistory}) => {
+
+	componentDidUpdate: {
+		var ch = document.getElementsByClassName('command-history-list')[0]
+		if (ch) {
+			//Give the DOM time to process the DOM changes.
+			setTimeout( () => ch.scrollTop = ch.scrollHeight, 0)
+		}
```

Comment:
This is manually editing the DOM, right? Seems very un-idiomatic for a React app. Ideally, all your components should be pure functions of their props.